pkg/flare/config_check.go

```diff
@@ -144,5 +144,21 @@ func PrintConfig(w io.Writer, c integration.Config) {
 		state := fmt.Sprintf("dispatched to %s", c.NodeName)
 		fmt.Fprintln(w, fmt.Sprintf("%s: %s", color.BlueString("State"), color.CyanString(state)))
 	}
+	printReportingDisabledInfo(w, c)
 	fmt.Fprintln(w, "===")
 }
+
+func printReportingDisabledInfo(w io.Writer, c integration.Config) {
+	var msg string
+	if c.MetricsExcluded && c.LogsExcluded {
+		msg = "Metrics and logs reporting disabled"
+	} else if c.MetricsExcluded {
+		msg = "Metrics reporting disabled"
+	} else if c.LogsExcluded {
+		msg = "Logs reporting disabled"
```

Comment:
wdyt about `This configuration matched a logs|metrics container-exclusion rule, it will not be run by the Agent.`

Reply:
I changed a bit this part as we discussed offline. Now it takes `IsCheckConfig()` and `IsLogConfig()` into account. I adapted the tests to this change too.